modules/usage/main.tf

```diff
@@ -0,0 +1,16 @@
+data "aws_s3_bucket" "usage" {
+  bucket = "${var.usage_s3_bucket_id}"
+}
+
+data "template_file" "policy_usage" {
+  template = "${file("${path.module}/policy-usage.json.tpl")}"
+
+  vars {
+    usage_s3_bucket_arn = "${data.aws_s3_bucket.usage.arn}"
```

Comment:
Previously we had the usage bucket ARN as a variable, now we're also requiring the bucket id/name to be passed in (instead of using the existing ARN). Is there a way to just use the existing ARN so they don't have to specify the same input parameter twice?

Reply:
I was looking at how to do that also.  The only way I found is the parse the bucket name out of the arn.  Which I can do.  They have a way to look up the arn by using the id, but not a way to look up the id from the arn.

Reply:
do you actually need just the bucket name though? it looks like you're only ever using the ARN?